src/Controls/tests/DeviceTests/Elements/Accessibility/AccessibilityTests.cs

```diff
@@ -0,0 +1,208 @@
+		public async Task ExcludedWithChildren(bool result)
+		{
+			var button = new Button();
+			AutomationProperties.SetExcludedWithChildren(button, result);
+			var excluded = await GetValueAsync<bool, ButtonHandler>(button, handler => button.IsExcludedWithChildren());
+			Assert.Equal(result, excluded);
+		}
+#endif
+
+		[Category(TestCategory.Accessibility)]
+		[Collection(ControlsHandlerTestBase.RunInNewWindowCollection)]
+		public class InNewWindowCollection : ControlsHandlerTestBase
+		{
+			[Fact]
+			public async Task ValidateIsImportantForAccessibility()
+			{
+				EnsureHandlerCreated(builder =>
+				{
+					builder.ConfigureMauiHandlers(handlers =>
+					{
+						handlers.AddMauiControlsHandlers();
+						handlers.AddHandler(typeof(Window), typeof(WindowHandlerStub));
+					});
+				});
+
+				// Ideally this would use a theory and class data for each control type.
+				//
+				// To accurately test this we need to attach the elements to the UI Tree.
+				// Currently tests that attach to the UI Tree can't run in parallel so it slows everyone down.
+				// So, I opted to just attach a layout and then iterate over the controls.
+				// I left the test case class in case it makes sense to switch in the future. 
+
+				var layout = new VerticalStackLayout();
+				var contentPage = new ContentPage()
+				{
+					Content = new ScrollView()
+					{
+						Content = layout
+					}
+				};
+
+				var expectedResults = new Dictionary<IView, bool>();
+
+				foreach (var control in new AllControlsTestCase())
+				{
+					var view = (IView)Activator.CreateInstance((Type)control[0]);
+
+					if (view is RefreshView rv)
+						rv.Content = new VerticalStackLayout();
+
+					if (view is VisualElement ve)
+					{
+						ve.WidthRequest = 50;
+						ve.HeightRequest = 50;
+					}
+
+					layout.Add(view);
+					expectedResults.Add(view, (bool)control[1]);
+				}
+
+				expectedResults.Add(contentPage, false);
+
+				var stringBuilder = new StringBuilder();
+
+				await CreateHandlerAndAddToWindow<IWindowHandler>(contentPage, async (handler) =>
+				{
+					// make sure everything has finished laying out
+					await Task.Delay(100);
+					foreach (var expected in expectedResults)
+					{
+						var actual = expected.Key.IsAccessibilityElement();
+						if (expected.Value != actual)
+						{
+							stringBuilder.AppendLine($"{expected.Key} Expected: {expected.Value} Actual: {actual}");
+						}
+					}
+				});
+
+				Assert.True(stringBuilder.Length == 0, stringBuilder.ToString());
+			}
+		}
+
+		class LabelWithText : Label
+		{
+			// on iOS UILabel will only be an accessible element if it has text
+			public LabelWithText()
+			{
+				Text = "testing";
+			}
+		}
+
+		class AllControlsTestCase : IEnumerable<object[]>
+		{
+			public IEnumerator<object[]> GetEnumerator()
+			{
+#if IOS
+				yield return new object[] { typeof(ActivityIndicator), false };
+#else
+				yield return new object[] { typeof(ActivityIndicator), true };
+#endif
```

Comment:
Are all of these if/defs based on your observations of the current iOS default platform-specifci behaviors?

Reply:
Yes, I also tested each of these in a basic Xcode project to verify the same behavior.